client/src/helpers/test/ClassList.spec.js

```diff
@@ -0,0 +1,52 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils'
+import ClassList from '../../outside/components/ClassList.vue'
+import VueRouter from 'vue-router'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import {
+  faDownload,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faDownload)
+
+const localVue = createLocalVue()
+
+const router = new VueRouter()
+localVue.use(VueRouter)
+localVue.component('fas-icon', FontAwesomeIcon)
+
+const wrapper =  shallowMount(ClassList, {
+  localVue,
+  router
+})
+
+
+describe('ClassName test', () => {
+  // let wrapper
+
+  describe('Basic testing', () => {
+    test('Mounts properly', () =>{
+      expect(wrapper.isVueInstance()).toBeTruthy()
+    })
+    test('renders as expected', () => {
+      expect(wrapper.html()).toMatchSnapshot()
+    })
+    const button = wrapper.find('.assignmentascsv')
+    test('has "Download Evaluation Notes as CSV button"', () => {
+      expect(button.text()).toContain(
+        'Download all assignment evaluation notes'
+      )
+    })
+    //   test('does clicking the button show the modal', () => {
```

Comment:
@bryan-gilbert 
This is the part I asked you about unit tests in general, I am not sure what should be tested when it comes to the export as csv button functionality. I could definitely use some feedback on it! Thank you!

Reply:
this is good enough for now.